lib/twilio.rb

```diff
@@ -0,0 +1,27 @@
+require 'twilio-ruby'
+
+
+class Message
+
+def send_message(message)
+
+  accountSID ='AC09169126869387fa7de484c4a6d71489'
+
+  auth_token = '0986e11fcbba9286f87ac57dab26f205'
+
+  @client = Twilio::REST::Client.new accountSID, auth_token
+
+  from = "+33644607099"
+
+  friends = { '+33661928033' => "David Parry"}
+
+    friends.each do |key, value|
+      message = @client.account.messages.create(
+          :from => from,
+          :to => key,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)